WPMedia/fusion-news-theme-blocks
# Tmedia 482 footer tap targets

## Description
Increase margin between links, increase tappability size to match mocks

Also used the mocks to find the actual target size of the links to match mocks https://app.zeplin.io/project/603fa53e2626ed1592e7c0e6/screen/603fa5c34a388fa3d696150e
<img width="965" alt="Screen Shot 2021-12-21 at 15 43 50" src="https://user-images.githubusercontent.com/5950956/147001446-f93e1725-0218-4f7f-ab28-5e50d31bfc04.png">

increased text size from 14px -> 16px and used rem. also increased line height to match mocks of surrounding elements per mocks

## Jira Ticket
- [TMEDIA-482](https://arcpublishing.atlassian.net/browse/TMEDIA-482)

## Acceptance Criteria
All footer links pass lighthouse tapability test 

## Test Steps

1. Checkout this branch `git checkout TMEDIA-482-footer-tap-targets`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/footer-block`
3. Go to http://localhost/pf/all-blocks/?_website=the-sun
4. Run lighthouse in the dev tools 
5. You should see the tappability rating for footers pass 

## Effect Of Changes
### Before

In the cdn lighthouse test: https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2Fcorecomponents-the-gazette-prod.cdn.arcpublishing.com%2Fall-blocks%2F

locally: 
![Screen Shot 2021-12-21 at 15 27 20](https://user-images.githubusercontent.com/5950956/146999884-8bab4c42-479e-4695-bdd4-41ed828aaca1.png)

### After
![Screen Shot 2021-12-21 at 15 26 03](https://user-images.githubusercontent.com/5950956/146999751-88bf6163-500b-4f52-995d-b41487b3e7f8.png)


## Dependencies or Side Effects
- The footer items are as intended by mocks now 

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored). -> not applicable
- [x] Confirmed relevant documentation has been updated/added. -> not applicable

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
